### DISCUSSION TITLE
iTwinId was not properly set up in the checkpoint

### DISCUSSION THREAD

**Discussion by dongluyang** [iTwin/itwinjs-core#4359] [Q&A]
when i follow the wiki to achieve a spa application, after I click login it through oauth2,
there is one remote rpc call failure, which blocks me go ahead, anyone can help me.
the error message is as belowed
{message: "iTwinId was not properly set up in the checkpoint", errorNumber: 65590,…}
errorNumber: 65590
isError: true
message: "iTwinId was not properly set up in the checkpoint"
name: "Validation Failed"
<img width="2524" alt="WeChat35002685ba1dea307cedf45d4e59f3b7" src="https://user-images.githubusercontent.com/22255561/192200484-726e48a5-2683-4246-aeb7-a247b7bfe63c.png">

**Comment by nick4598** [comment]
Hi, on the failed getConnectionProps request there should be a header "X-Correlation-Id". Could you post that here?
